config/envs/unit_test.py

```diff
@@ -57,3 +59,6 @@ class UnitTestConfig(DefaultConfig):
     # Which is overkill for now. 28/06/2024.
     CELERY = DefaultConfig.CELERY
     CELERY["task_always_eager"] = True
+
+    E2E_NOTIFY_FIND_API_KEY = os.environ.get("E2E_NOTIFY_FIND_API_KEY")
+    E2E_DEVTEST_BASIC_AUTH = os.environ.get("E2E_DEVTEST_BASIC_AUTH")  # Format: "username:password"
```

Comment:
Maybe think it's clearer to break our `USERNAME` and `PASSWORD` but not a deal-breaker!